lib/diagnostic.js

```diff
@@ -28,7 +28,7 @@ let person = {
 person.fullName();
 
 // Question 2:  What returns when we call person.fullName();
-let answer2 = null; // replace null with your response
+let answer2 = Ben Adamski; // replace null with your response
```

Comment:
This will cause an error.  Strings need to be wrapped in single quotation marks ('), double quotation marks("), or grave accents (`).